src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import Image from "next/image";
+
+import * as S from "./styles";
+
+const ContactUsContent: React.FC = () => {
+  const [name, setName] = useState<string | null>(null);
+  const [subject, setSubject] = useState<string | null>(null);
+  const [content, setContent] = useState<string | null>(null);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
+
+    if (name === null) {
+      setName("");
+    }
+
+    if (subject === null) {
+      setSubject("");
+    }
+
+    if (content === null) {
+      setContent("");
+    }
+
+    if (!name || !subject || !content) {
+      return;
+    }
+
+    const body =
+      `Dear Bernard,\n\n` +
+      `${content}\n\n` +
+      `Thank you.\n\n` +
+      `Best regards,\n\n` +
+      `${name}\n`;
+
+    location.href =
+      `mailto:bernardsw99@gmail.com` +
+      `?cc=krishna.bernardus@gmail.com` +
+      `&subject=${encodeURIComponent(subject)}` +
+      `&body=${encodeURIComponent(body)}`;
+  };
+
+  return (
+    <div className={`flex flex-col items-center overflow-hidden`}>
+      <div className = {S.MyDescription}>
```

Comment:
LMAO linter nya g jalan, try to set up the linter and the setting in vscode biar dia autolint everytime lu save (coba cek google)

Reply:
ok